content/documentation/tool-platform/content.md

```diff
@@ -0,0 +1,47 @@
++++
+fragment = "content"
+weight = 20
+
+title = "Eclipse Theia: Development Tool Platform"
+
+[sidebar]
+  sticky = true
++++
+
+For creating a specialized C/C++ tool or custom IDE, it is typically most efficient to build on an existing tool platform, which already provides generic tool features, such as a file explorer, editors with rich language-editing support, menus, user interfaces for process output or debugging, etc.
+To meet the specific requirements of your particular tool or IDE, you then extend and customize this generic platform with specific extensions.
+
+<p align="center" style="padding-top: 1em; padding-bottom: 1em">
+  <img src="/images/cdtcloud-screenshot.png" width="85%" style="border-radius: 5px" />
+</p>
+
+As you usually want to have full control over every aspect of your custom tool, [Eclipse Theia™](https://theia-ide.org) is a popular choice for a tool platform in general, and for custom C/C++ development in particular.
```

Comment:
I think we should also mention VS Code and that many but not all of the CDT.cloud components can be used there.